smart_open/tests/test_bytebuffer.py

```diff
@@ -9,23 +9,25 @@
 import random
 import unittest
 
-import six
-
 import smart_open.bytebuffer
 
 
 CHUNK_SIZE = 1024
 
 
+def int2byte(i):
+    return bytes((i, ))
+
+
 def random_byte_string(length=CHUNK_SIZE):
-    rand_bytes = [six.int2byte(random.randint(0, 255)) for _ in range(length)]
+    rand_bytes = [int2byte(random.randint(0, 255)) for _ in range(length)]
```

Comment:
non-determenistic tests, not sure then this is a good idea

Reply:
Yeah, I agree that having random in the tests is a bit of a code smell, but it appears to be a core feature of those tests. I'm not sure how we'd go about removing them. Let's handle that in a separate PR.